runner/ansible/roles/2.1.3/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.1.3'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # SAP HANA Resource Agent is not configured, then pass
+    cibadmin -Q --xpath "//primitive[@type='SAPHana']/@type" || exit 0
+    # if configured and not msl, then fail
+    cibadmin -Q --xpath "//master/primitive[@type='SAPHana']/@type" || exit 1
+    # check the following values, if any do not match expected fail.
+    XPATH="//primitive[@type='SAPHana']/operations/op"
+    [[ $(cibadmin -Q --xpath "${XPATH} [@name='monitor'] [@role='Master']" | grep -oP 'interval="\K[^"]+') != "{{ expected[test_id + '.master_interval'] }}" ]] && exit 1
+    [[ $(cibadmin -Q --xpath "${XPATH} [@name='monitor'] [@role='Master']" | grep -oP 'timeout="\K[^"]+') != "{{ expected[test_id + '.master_timeout'] }}" ]] && exit 1
+    [[ $(cibadmin -Q --xpath "${XPATH} [@name='monitor'] [@role='Slave']" | grep -oP 'interval="\K[^"]+') != "{{ expected[test_id + '.slave_interval'] }}" ]] && exit 1
+    [[ $(cibadmin -Q --xpath "${XPATH} [@name='monitor'] [@role='Slave']" | grep -oP 'timeout="\K[^"]+') != "{{ expected[test_id + '.slave_timeout'] }}" ]] && exit 1
+    [[ $(cibadmin -Q --xpath "${XPATH} [@name='start']" | grep -oP 'interval="\K[^"]+') != "{{ expected[test_id + '.start_interval'] }}" ]] && exit 1
+    [[ $(cibadmin -Q --xpath "${XPATH} [@name='start']" | grep -oP 'timeout="\K[^"]+') != "{{ expected[test_id + '.start_timeout'] }}" ]] && exit 1
+    [[ $(cibadmin -Q --xpath "${XPATH} [@name='stop']" | grep -oP 'interval="\K[^"]+') != "{{ expected[test_id + '.stop_interval'] }}" ]] && exit 1
+    [[ $(cibadmin -Q --xpath "${XPATH} [@name='stop']" | grep -oP 'timeout="\K[^"]+') != "{{ expected[test_id + '.stop_timeout'] }}" ]] && exit 1
+    [[ $(cibadmin -Q --xpath "${XPATH} [@name='promote']" | grep -oP 'interval="\K[^"]+') != "{{ expected[test_id + '.promote_interval'] }}" ]] && exit 1
+    [[ $(cibadmin -Q --xpath "${XPATH} [@name='promote']" | grep -oP 'timeout="\K[^"]+') != "{{ expected[test_id + '.promote_timeout'] }}" ]] && exit 1
+    XPATH="//primitive[@type='SAPHana']/instance_attributes/nvpair"
```

Comment:
I would add a new line here, to add visibility

Reply:
done